bionic/Dockerfile

```
@@ -0,0 +1,38 @@
+FROM ubuntu:bionic
+
+LABEL maintainer="luca.heltai@gmail.com"
+
+USER root
+RUN apt-get update && apt-get install -y software-properties-common \
+    && add-apt-repository ppa:ginggs/deal.ii-backports \
+    && apt-get update && apt-get install -y \
+    locales \
+    git \
+    ssh \
+    wget \
+    ninja-build \
+    numdiff \
+    libdeal.ii-dev \
```

The beauty about this string of commands is the fact that it will happily work for a number of different ubuntu releases (including bionic and focal). But there is one caveat - the moment we add a new version to the ppa you will install 9.2.0 intsead of 9.1.1. Would you mind future proofing this and specify the precise version?
```suggestion
    libdeal.ii-dev=9.1.1-9build2 \
```